src/docs/get-started/install/_get-sdk-win.md

```diff
@@ -9,7 +9,7 @@
     [SDK releases][] page.
  1. Extract the zip file and place the contained `flutter`
     in the desired installation location for the Flutter SDK
-    (for example, `C:\src\flutter`).
+    (for example, `C:\Users\Documents`).
```

Comment:
For me this would be something like `C:\Users\Jonah\Documents`, this would be for a user named 'Documents'. Is there a pattern for pointing the user towards this instead?

Reply:
D'OH. Of course. Fixed.